actionpack/test/controller/force_ssl_test.rb

```diff
@@ -92,6 +92,22 @@ def cheeseburger
   end
 end
 
+class RedirectToSSLIfSessionStoreDisabled < ForceSSLController
```

Comment:
This is testing an implementation detail. Can we test it at integration level so we don't need to mock the request object?